metainfo-rename.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/data/eu.lucaweiss.lpa_gtk.metainfo.xml b/data/eu.lucaweiss.lpa-gtk.metainfo.xml
+similarity index 96%
+rename from data/eu.lucaweiss.lpa_gtk.metainfo.xml
+rename to data/eu.lucaweiss.lpa-gtk.metainfo.xml
+index 15cbc53..bb67209 100644
+--- a/data/eu.lucaweiss.lpa_gtk.metainfo.xml
++++ b/data/eu.lucaweiss.lpa-gtk.metainfo.xml
+@@ -4,7 +4,7 @@ SPDX-FileCopyrightText: 2024 Luca Weiss
+ SPDX-License-Identifier: CC-BY-SA-4.0
+ -->
+ <component type="desktop-application">
+-    <id>eu.lucaweiss.lpa_gtk</id>
++    <id>eu.lucaweiss.lpa-gtk</id>
+ 
+     <name>eSIM Manager</name>
+     <summary>Download and manage eSIM profiles</summary>
+diff --git a/data/meson.build b/data/meson.build
+index 6afca15..6aaa738 100644
+--- a/data/meson.build
++++ b/data/meson.build
+@@ -14,7 +14,7 @@ if ascli_exe.found()
+   )
+ endif
+ 
+-install_data('eu.lucaweiss.lpa_gtk.metainfo.xml',
++install_data('eu.lucaweiss.lpa-gtk.metainfo.xml',
```

Comment:
you don't need to do that. You can put `rename-appdata-file` in the manifest if really needed. But then as mentionned prerviously, why using a different id?

Reply:
Removed.